### PR TITLE
Set up testing environment for Client

### PR DESCRIPTION
**Pull request by Carltesio**
# [PT Story](https://www.pivotaltracker.com/n/projects/2439202/stories/171878948)
## Changes proposed in this pull request:
* Adds cypress for acceptance testing
* Add cy:open to run cypress and yarn server
* set base url in Cypress json
* Configure Enzyme for jest to run components tests

## What I have learned working on this feature:
* To change the local host port

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
```
Comment (ghost) [inline cypress/plugins/index.js:3]: Can you remove this commented section, thanks

Comment (hmvitous) [reply cypress/plugins/index.js:3]: I was thinking the same about commented code

Comment (Carltesio) [reply cypress/plugins/index.js:3]: Comments removed.

**Comment by hmvitous** [comment]
I say its ready to merge

**Comment by ghost** [comment]
OK. You may now merge

File: cypress/plugins/index.js (hunk 1 above)
Comment (tochman) [reply cypress/plugins/index.js:3]: I would advise against that.

**Pull request merged**